### PR TITLE
Version 0.2

### PR DESCRIPTION
**Pull request by AHReccese**
#### Reference Issues/PRs
#16 
#### What does this implement/fix? Explain your changes.
Version 0.2 with the full support of scikit NN models prepared.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -22,7 +22,26 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- `LabelBinerizer` Transporter updated
+- `linear model` chain updated
+
 
-[Unreleased]: https://github.com/openscilab/pymilo/compare/v0.1...dev
+
+[Unreleased]: https://github.com/openscilab/pymilo/compare/v0.2...dev
 [0.1]: https://github.com/openscilab/pymilo/compare/e887108...v0.1
+[0.2]: https://github.com/openscilab/pymilo/compare/v.0.1...v0.2
+
```
Comment (sadrasabouri) [inline CHANGELOG.md:46]: You violated the pattern. Try looking at `release` PRs in pycm.

**Review by sadrasabouri — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -120,10 +120,11 @@ Pymilo is an open source Python package that provides a simple, efficient, and s
 | scikit-learn | PyTorch | 
 | ---------------- | ---------------- | 
 | Linear Models &#x2705; | - | 
-| Neural networks &#x274C; | -  | 
+| Neural networks &#x2705; | -  | 
 | Clustering &#x274C; | -  | 
 | Trees &#x274C; | -  | 
 | Ensemble Models &#x274C; | - | 
+Details are available in `supported-models.md` file.
```
Comment (sepandhaghighi) [inline README.md:127]: Details are available in [Supported Models](https://github.com/openscilab/pymilo/blob/main/SUPPORTED_MODELS.md).

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (AHReccese) [reply CHANGELOG.md:46]: Try to use a better word than violation.
Ok sure, I will recheck the details.
Surly it wasn't intentional.

File: README.md (hunk 2 above)
Comment (AHReccese) [reply README.md:127]: My bad, sure.

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,23 +6,43 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+- `linear model` chain updated
 - GeneralDataStructure transporter enhanced
 - LabelBinerizer transporter updated
 - transporters' chain router added to `pymilo func`
 - NeuralNetwork params initialized in `pymilo_param`
 - `pymilo_test` updated to support multiple models
 - `linear_model_chain` refactored
+
+
+[Unreleased]: https://github.com/openscilab/pymilo/compare/v0.2...dev
```
[the remark this answers is not in the document]
Comment (AHReccese) [reply CHANGELOG.md]: my bad bad!
done.

**Review by sepandhaghighi — APPROVED** [review]
🚀

**Pull request merged**